jnosql-communication/jnosql-communication-column/src/main/java/org/eclipse/jnosql/communication/column/Column.java

```diff
@@ -67,18 +64,17 @@ public interface Column extends Entry {
      * @param value - column's value
      * @param <V>   the value type
      * @return a column instance
-     * @throws NullPointerException when there is any null parameter
+     * @throws NullPointerException when name is null
      * @see Columns
      */
     static <V> Column of(String name, V value) {
         Objects.requireNonNull(name, "name is required");
-        Objects.requireNonNull(value, "value is required");
         return new DefaultColumn(name, getValue(value));
     }
 
     private static Value getValue(Object value) {
         if (value instanceof Value) {
```

Comment:
@otaviojava, is this place a good one to apply Pattern Matching, isn't it?

Reply:
At the end, we still need the return; thus, it does not change much.